ingest/expression_files/mtx.py

```diff
@@ -116,6 +151,23 @@ def check_duplicates(names: List, file_type: str):
             raise ValueError(msg)
         return True
 
+    @staticmethod
+    def get_data_start_line_number(file_path: str) -> int:
+        """ Determines what line number data starts.
+
+         Returns
+         ----------
+            i (int) - Line number where data starts
+        """
+        i = 1
+        for line in file_path:
+            if line.startswith("%"):
+                i = +1
+            else:
+                # First line w/o '%' is mtx dimension. So skip this line (+2)
+                return i + 2
+        raise ValueError("MTX file did not contain data")
```

Comment:
Blocker: this user-facing messages needs refinement.  Please:

1. Use a term or phrase more descriptive than only "data"
2. Make the error message more actionable by the user.  How can they fix it?

Reply:
b625656

Reply:
e433046